### ISSUE TITLE
Registered User

### ISSUE THREAD

**Issue by antiproton** [jsonchess/lightsquare#108]
it would be possible to implement a user profile. With Avatar, country, birthday, etc.?
Best Regards R. U.

**Comment by gushogg-blake** [comment]
Yes I think this is something that should be added eventually, although one of the goals of Lightsquare is to put most of the focus on the games.  But I agree that some more personal details about the users would make it better.  My main priority is big missing features like variants and tournaments.  Feel free to try out some ideas for a profile feature, and make a pull request.

**Comment by antiproton** [comment]
Yes!
http://www.genialchess.de
Best Regards R. U.
